INV4/pallet-ipt/src/benchmarking.rs

```diff
@@ -0,0 +1,64 @@
+//! Benchmarks for IPT Pallet
+#![cfg(feature = "runtime-benchmarks")]
+
+pub use super::*;
+use frame_benchmarking::{
+    account, benchmarks, impl_benchmark_test_suite, vec, whitelisted_caller, Box,
+};
+use frame_system::RawOrigin;
+use primitives::SubIptInfo;
+use sp_io::hashing::blake2_256;
+
+const SEED: u32 = 0;
+
+benchmarks! {
+    mint {
+        let s in 0 .. 100;
+        let caller: T::AccountId = whitelisted_caller();
+        let amount: <T as pallet::Config>::Balance = 300u32.into();
+        let target: T::AccountId = account("target", 0, SEED);
+    }: _(RawOrigin::Signed(caller), (T::IptId::from(s), None), amount, target)
+
+    burn {
+        let s in 0 .. 100;
+        let caller: T::AccountId = whitelisted_caller();
+        let amount: <T as pallet::Config>::Balance = 300u32.into();
+        let target: T::AccountId = account("target", 0, SEED);
+
+        Pallet::<T>::mint(RawOrigin::Signed(caller.clone()).into(), (T::IptId::from(s), None), amount, target.clone())?;
+    }: _(RawOrigin::Signed(caller), (T::IptId::from(s), None), amount, target)
+
+    operate_multisig {
+        let s in 0 .. 100_000;
+        let caller: T::AccountId = whitelisted_caller();
+        let call: <T as pallet::Config>::Call = frame_system::Call::<T>::remark {
+            remark: vec![0; s as usize],
+        }.into();
+    }: _(RawOrigin::Signed(caller), false, (T::IptId::from(s), None), Box::new(call))
+
+    vote_multisig {
+        let s in 0 .. 100_000;
+        let caller: T::AccountId = whitelisted_caller();
+        let call: <T as pallet::Config>::Call = frame_system::Call::<T>::remark {
+            remark: vec![0; s as usize],
+        }.into();
+    }: _(RawOrigin::Signed(caller), (T::IptId::from(s), None), blake2_256(&call.encode()))
+
+    withdraw_vote_multisig {
+        let s in 0 .. 100_000;
+        let caller: T::AccountId = whitelisted_caller();
+        let call: <T as pallet::Config>::Call = frame_system::Call::<T>::remark {
+            remark: vec![0; s as usize],
+        }.into();
+    }: _(RawOrigin::Signed(caller), (T::IptId::from(s), None), blake2_256(&call.encode()))
```

Comment:
Same as above + this vote qas never made

Reply:
Check